themes/mms-cloud/page.html

```diff
@@ -50,9 +50,8 @@
       </button>
       <span class="alert-message">
 
-        <h4 class="alert-heading" style="margin-bottom: 10px;">Change is coming soon to the MongoDB Agents!</h4>
-        Monitoring, Backup and Automation are merging to become
-        the MongoDB Agent.
+        <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has been released!</h4>
+        Monitoring, Backup and Automation are now the MongoDB Agent.
```

Comment:
This doesn't make sense if you haven't seen the previous alert and fill in that we're referring to the old agents. It might be practical to clarify with something like `The Monitoring, Backup, and Automation Agents have been replaced by the MongoDB Agent.` Or something.

Reply:
Agreed. Fixed.